src/io/dss/dss2eng.jl

```diff
@@ -725,6 +734,21 @@ function _dss2eng_transformer!(data_eng::Dict{String,<:Any}, data_dss::Dict{Stri
             end
         end
 
+        # Adds regcontrol to transformer if present
+        if !isempty(reg_obj) && (haskey(reg_obj,"$id") || (haskey(dss_obj,"bank") && haskey(reg_obj,"$(dss_obj["bank"])")))
+            reg_ctrl = haskey(reg_obj,"$id") ? reg_obj["$id"] : reg_obj["$(dss_obj["bank"])"]
+            defaults = _apply_ordered_properties(_create_regcontrol(id; _to_kwargs(reg_ctrl)...), reg_ctrl)
+            
+            eng_obj["controls"] = Dict{String,Any}(
+                "name" => reg_ctrl["name"],
+                "vreg" => defaults["vreg"],
+                "band" => defaults["band"],
+                "ptratio" => defaults["ptratio"],
+                "ctprim" => defaults["ctprim"],
+                "r" => defaults["r"],
+                "x" => defaults["x"]
```

Comment:
these need to be updated to vectors as in the documentation, using the `dss_obj["winding"]` to determine which winding is being controlled. Also, probably `tm_fixed` on the transformer should be updated according to which winding is being controlled?